assets/js/payment-method-extensions/payment-methods/paypal/index.js

```diff
@@ -39,6 +40,13 @@ const paypalPaymentMethod = {
 	),
 	content: <Content />,
 	edit: <Content />,
+	icons: [
+		{
+			id: 'paypal',
+			alt: __( 'PayPal', 'woo-gutenberg-products-block' ),
+			src: `${ WC_BLOCKS_ASSET_URL }img/paypal-mark.jpg`,
+		},
+	],
```

Comment:
so feasibly, this payment method could just do something like this right?

```js
icons: [ 'paypal' ]
```

or:

```js
icons: [ { id: 'paypal' } ],
```

Reply:
Only if we include `paypal` in the common methods; should I add this as a common method too?

Reply:
If this is the mark for paypal standard? Maybe make the icon a bit more specific `paypal-standard` in that case?

Reply:
PayPal has no specific marks. PayPal does also support credit cards but not clear which. Do you think we should leave out the marks for PayPal entirely?

Reply:
sure, I think it was only added here since it might get used with the cart icon output right?But per the recent slack convo (C8X6Q7XQU-slack-C8X6Q7XQU) maybe we don't want paypal output with the cart icons? I'm okay with rolling without paypal either as a common icon or implemented for cart.

Reply:
Let's kill it with 🔥